.github/BECOMING_A_CORE_CONTRIBUTOR.md

```diff
@@ -38,7 +38,7 @@ Here, we describe general expectations from core contributors:
 
 ### Pull Requests (PRs)
 
-- Pull requests are the evolutionary mechanism of Lightning, so quality is extremely important. Make sure contributors adhere to the guidelines described in the [contributing section](CONTRIBUTING.md).
+- Pull requests are the evolutionary mechanism of Lightning, so quality is extremely important. Make sure contributors adhere to the guidelines described in the [contributing section](CONTRIBUTING.md#pull-requests-prs).
```

Comment:
```suggestion
- Pull requests are the evolutionary mechanism of Lightning, so quality is extremely important. Make sure contributors adhere to the guidelines described in the [contributing section](CONTRIBUTING.md#pull-request).
```

The section is called `Pull Request`, not `Pull Requests PRs` (https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md#pull-request)

Reply:
You are right I mess with the other in BECOMING_A_CORE_CONTRIBUTOR.md